src/go/k8s/Makefile

```diff
@@ -0,0 +1,101 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= gcr.io/vectorized/controller:latest
```

Comment:
nit: should it maybe have k8s in the name of the image? I find controller a bit too generic.

Reply:
I agree. I propose `redpanda-k8s-operator`. 

The public release will be available through docker hub not GCR, right @senior7515?

Reply:
I like that name!

Reply:
Ok. Will change in the next push.